### DISCUSSION TITLE
Problems on scene update.

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#821] [Q&A] [answered]
Hi Marco,
I use flagpost to show some information.
Due to location changes, the flagpost need be update. I use plotter.remove(old flagpost) and then creat new one at new location but the old one is still there.
Here is the screenshot.
![image](https://user-images.githubusercontent.com/116484589/221407727-8ad8142b-a42a-479d-a3a9-9a116da8ae39.png)
Can the old flagpost be deleted?  The "del old_flagpost" doesnt work.
BTW, when I change the parameter c to other non-default value the text could not be seen. I suppose the reason is it controls both text and box colors according api doc. Maybe you can have a look. :)

**Comment by ZiguoAtGitHub** [comment, marked as answer]
One more thing:
I tried using pos() to update the position of the flagpost and related actors (sphere, line and circle).
Sphere and circle are moved to correct location but the flagpost doesnt be moved and the line is moved to wrong location.
Here is the screenshot.
![image](https://user-images.githubusercontent.com/116484589/221410578-cc99c8db-25f1-4e99-bbac-65202915b929.png)
Hope it helpful. :)

**Reply by marcomusy** [reply]
Hi I just pushed a new version with the fix, hopefully it will work.
Note that now you don't need anymore to add/remove the same flagpost - you can simply update its position and content.
Check out the new example:
https://github.com/marcomusy/vedo/blob/master/examples/basic/mousehover0.py

**Reply by ZiguoAtGitHub** [reply]
Awesome!!
The Line position updated with pos() is also fixed?
Thanks.

**Reply by marcomusy** [reply]
that should be fixed too.. I could not fix the coloring.. there must be some upstream issue there with vtk.. i will try again when i have more time..

**Reply by ZiguoAtGitHub** [reply]
It sounds more patience needed for fixing the coloring problem. :)
Actually, the default colors are good enough for my case. So you have enough time and I have enough patience. 😁
Thanks again for your wonderful work!
